src/views/virtualmachines/details/components/details/utils/gridHelper.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+
+import { NodeModel, PodModel } from '@kubevirt-ui/kubevirt-api/console';
+import {
+  V1VirtualMachineInstance,
+  V1VirtualMachineInstanceGuestAgentInfo,
+} from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { K8sResourceCommon, ResourceLink } from '@openshift-console/dynamic-plugin-sdk';
+
+import FirstItemListPopover, {
+  getVMIIPAddresses,
+} from '../../../../list/components/FirstItemListPopover/FirstItemListPopover';
+import MutedTextDiv from '../components/MutedTextDiv/MutedTextDiv';
+
+// import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getVMIPod } from './vmiHelper';
+
+export type VirtualMachineDetailsRightGridLayoutPresentation = {
+  pod: React.ReactNode;
+  ipAddress: React.ReactNode;
+  hostname: React.ReactNode;
+  timezone: React.ReactNode;
+  node: React.ReactNode;
+  userCredentials: React.ReactNode;
+  sshAccess: React.ReactNode;
+};
+
+export const getStoppedVMRightGridPresentation =
+  (): VirtualMachineDetailsRightGridLayoutPresentation => {
+    const notAvailable = <MutedTextDiv text="Not available" />;
+    const virtualMachineIsNotRunning = <MutedTextDiv text="Virtual machine is not running" />;
```

Comment:
Would it be possible to pass `t` to this function so you can translate these strings?

Reply:
sure, added :)